queries.sql

```diff
@@ -52,4 +25,25 @@ SELECT full_name, name FROM animals RIGHT JOIN owners ON owners.id = animals.own
 SELECT COUNT(*), species.name AS species_name FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
 SELECT animals.name AS animal_name, species.name AS species_name, full_name FROM animals INNER JOIN species ON species.id = animals.species_id INNER JOIN owners ON owners.id = animals.owner_id WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
 SELECT * FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE escape_attempts = 0 AND full_name = 'Dean Winchester';
-SELECT full_name, COUNT(animals.name) AS animal_count FROM owners INNER JOIN animals ON owners.id = animals.owner_id GROUP BY full_name;
\ No newline at end of file
+SELECT full_name, COUNT(animals.name) AS animal_count FROM owners INNER JOIN animals ON owners.id = animals.owner_id GROUP BY full_name;
+
+/* DAY 4 */
+SELECT animals.name, vets.name, date_of_visit FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'William Tatcher' ORDER BY date_of_visit DESC LIMIT 1;
+SELECT COUNT(*), vets.name FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Stephanie Mendez' GROUP BY vets.name;
+SELECT vets.name, species.name FROM vets LEFT JOIN specializations ON vets.id = specializations.vet_id LEFT JOIN species ON specializations.species_id = species.id;
+SELECT animals.name, vets.name, date_of_visit FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Stephanie Mendez' AND date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+SELECT animals.name, COUNT(visits.vet_id) FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id GROUP BY animals.name;
+SELECT animals.name, vets.name FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id WHERE vets.name = 'Maisy Smith' ORDER BY date_of_visit ASC LIMIT 1;
+SELECT animals.name, animals.date_of_birth, animals.escape_attempts, animals.neutered, animals.weight_kg, animals.species_id, animals.owner_id, vets.name, vets.age, vets.date_of_graduation FROM animals INNER JOIN visits ON animals.id = visits.animal_id INNER JOIN vets ON vets.id = visits.vet_id;
+
```

Comment:
Please ensure all commented code are removed from your codebase to make it cleaner 👍🏼